### ISSUE TITLE
Errors with the updated version

### ISSUE THREAD

**Issue by Kaypark93** [satrajitgithub/tumor_classifier_3d#1]
[INFO] Loading models from:
models/modelSegmenter_T1c+T2+FLAIR_ep203_dice_3.5877_vloss0.1743.h5
From /home/usr/parkk/.local/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From /home/usr/parkk/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3733: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From /home/usr/parkk/.local/lib/python3.6/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
